Core1Platform/01a_FieldsOfApplicaton.md

```diff
@@ -0,0 +1,37 @@
+The average age of running ERP solutions are about 12 years. This means in most cases that you are not able to quickly follow new evolutions in e-commerce with your legacy ERP system. Changing of processes and data models are then extremely time-consuming and expensive. Additionally, changes may be risky and can have a bad impact on business transactions that are stable.   
+With the *Actindo Core1 Platform* you can loosely couple a D2C (direct-to-consumer) infrastructure to your legacy IT. The digital operations platform with its core orchestration processes the interaction between channels, 3PL (third-party logistics), PSP (payment service provider), and your legacy ERP. For example, you can exchange the following data:
+- Material numbers and article master data
+- Invoices and credit notes   
+
+The direction of the data flow is thereby determined individually by the process orchestration. Your existing IT infrastructure remains untouched. 
+The *Actindo Core1 Platform* supports the transformation to a D2C business with all benefits that provides running an own branded online shop. You can use the *Actindo Core1 Platform* to modernize your e-commerce, to stay up-to-date, and to implement new evolutions.  
+
+
+## Outdated and unsustainable ERP system in use
+
+Your company is already established in the market and in B2C (business-to- consumer) business, but your IT infrastructure is completely outdated and not maintainable, because the legacy IT is self-built and highly customized.   
+In this case, you can use the *Actindo Core1 Platform* to replace the existing infrastructure in waves. Actindo analyzes your specific needs and designs modern and useful processes. The process orchestration then makes your processes transparent and removes single points of failure.
+
+
+## Young start-up
+
+Your company is going to take off, and you expect significant growth. Your current tools will no longer be able to handle your business. You can run the *Actindo Core1 Platform* as a standalone platform that quickly solve specific pain points in first waves. This enables seamless transition without any downtimes. And then expand to a lean IT landscape with a digital operations platform in its core. The digital operations platform allows you to adjust processes continously and to connect to new services and channels that are needed.   
+The wide range of classic ERP features helps to digitize, automate, and optimize e-commerce processes. You can use it as all-in-one-solution to do the following:   
+ - Maintain your business data.   
+ - Implement and monitor your e-commerce processes.   
+ - Get, transform, and transfer data from service providers, shops, and fulfillment providers.   
+- Reporting on products, individual KPIs, and statistics.    
+- And much more...
+
+
+## Conclusion
```

Comment:
Vielleicht ein schönerer/expressiver Header? Z.B. Interlocking flexible solution / All-in-one solution... (Keine Ahnung, marketing-ähnlich)